spec/runner_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/ship'
+require './lib/cell'
+require './lib/board'
+require './lib/battleship_runner'
+
+RSpec.describe 'runner' do
+
+end
+
+#set up rspec
+#randomize computer placement
+#is the rubber duckie cool?
+# do we need turn class or does getschomp break up script?
```

Comment:
These are great questions. I am looking forward to getting to the bottom of this.